### ISSUE TITLE
galaxy_info is a required property despite meta-no-info

### ISSUE THREAD

**Issue by tumbl3w33d** [ansible/ansible-lint#2469]
##### Summary

Since 6.6.0 my roles with meta/main.yml cause warnings although I explicitly [opted out of the galaxy stuff](https://github.com/ansible/schemas/issues/163) using `meta-no-info` in the skip_list. However, I don't want to opt out of schema[meta] validation in general.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible-lint 6.6.0 using ansible 2.13.4
```

- ansible installation method: pip 
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

`5.15.65-1-MANJARO #1 SMP PREEMPT Mon Sep 5 10:15:47 UTC 2022`

##### STEPS TO REPRODUCE

**.ansible-lint**
```yaml
skip_list:
  - meta-no-info
```
Example of a role with simple meta dependency defined in meta/main.yml
```yaml
---
dependencies:
  - role: other_role
```

##### Desired Behavior

`meta-no-info` should cover this. No warning should appear.

##### Actual Behavior

``` console
schema: 'galaxy_info' is a required property (schema[meta])
myrole/meta/main.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``playbook-schema``,
``tasks-schema``.

This rule is not skippable and stops further processing of the file.

Schema bugs should be reported towards (https://github.com/ansible/schemas) project instead of ansible-lint.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
